source/config/configFlags.py

```diff
@@ -220,3 +220,23 @@ def _displayStringLabels(self):
 			# Translators: This is a label for the automatic update behaviour for add-ons.
 			self.DISABLED: _("Disabled"),
 		}
+
+@unique
+class OutputMode(DisplayStringIntEnum):
+	OFF = 0
+	SPEECH = 1
+	BRAILLE = 2
+	SPEECH_AND_BRAILLE = 3
+
+	@property
+	def _displayStringLabels(self):
+		return {
+			# Translators: This is a way of reporting information.
```

Comment:
Perhaps this is a bit clearer? can you update other strings to match

```suggestion
			# Translators: A label for an option to choose a method of reporting information, e.g. font attributes.
```